src/Routing/RouteBuilder.php

```diff
@@ -987,8 +987,10 @@ public function scope($path, $params, $callback = null)
             $params = [];
         }
         if (!is_callable($callback)) {
-            $msg = 'Need a callable function/object to connect routes.';
-            throw new InvalidArgumentException($msg);
+            throw new InvalidArgumentException('Need a callable function/object to connect routes.');
+        }
+        if (!is_array($params)) {
+            throw new InvalidArgumentException('Invalid params passed.');
```

Comment:
why we need to this Exception ?

Reply:
In case of invalid callback this sanity check seems to make sense with the current dirty input params.